lib/postgrex.ex

```diff
@@ -340,9 +341,29 @@ defmodule Postgrex do
     DBConnection.child_spec(Postgrex.Protocol, opts)
   end
 
-  ## Helpers
+  @doc """
+  Returns a stream on a prepared query
+
+  Stream consumes memory in chunks of size `max_rows` at most ( see options )
+
+  This is useful for processing _large_ datasets
+
+  ### options
 
+    * `:max_rows` - maximum numbers of rows backend will result ( default to `#{ @max_rows } )
+    * `:portal` -  Name of then underlying portal that will hold results, it will be generated unless provided
+    * `:pool_timeout` - Time to wait in the queue for the connection (default: `#{@pool_timeout}`)
+    * `:queue` - Whether to wait for connection in a queue (default: `true`)
+  """
+  @spec stream( conn, Postgrex.Query.t, list, Keyword.t ) :: Postgrex.Stream.t
```

Comment:
`DBConnection.t` instead of `conn` please as it can only be used inside transaction fun.